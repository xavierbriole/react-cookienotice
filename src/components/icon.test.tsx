import React from 'react'
import { render } from '@testing-library/react'

import Icon from './icon'

describe('Icon', () => {
  it('should render', () => {
    const { container } = render(<Icon />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
