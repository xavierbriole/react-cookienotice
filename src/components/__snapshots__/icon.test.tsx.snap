// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render 1`] = `
<div
  class="react-cookienotice-icon"
/>
`;
